cmake/ImpactXFunctions.cmake

```diff
@@ -80,11 +75,9 @@ macro(impactx_set_default_install_dirs)
     if(CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
         include(GNUInstallDirs)
         if(NOT CMAKE_INSTALL_CMAKEDIR)
-            set(CMAKE_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake"
-                    CACHE PATH "CMake config package location for installed targets")
+            set(CMAKE_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake")
             if(WIN32)
-                set(CMAKE_INSTALL_LIBDIR Lib
-                        CACHE PATH "Object code libraries")
+                set(CMAKE_INSTALL_LIBDIR Lib)
                 set_property(CACHE CMAKE_INSTALL_CMAKEDIR PROPERTY VALUE "cmake")
```

Comment:
CMake Error at cmake/ImpactXFunctions.cmake:81 (set_property):
  set_property could not find CACHE variable CMAKE_INSTALL_CMAKEDIR.  Perhaps
  it has not yet been created.
Call Stack (most recent call first):
  CMakeLists.txt:50 (impactx_set_default_install_dirs)